lib/routers/establishment/index.js

```diff
@@ -81,7 +86,12 @@ router.put('/:establishment',
   whitelist('name', 'address', 'procedure', 'breeding', 'supplying', 'authorisations'),
   validateEstablishment,
   updateDataAndStatus(),
-  update
+  submit('update')
+);
+
+router.put('/:establishment/grant',
+  permissions('establishment.update'),
```

Comment:
```suggestion
  permissions('establishment.update'),
  whitelist(),
```

Assuming the request body _should_ be empty on this call?

Reply:
Yeah it should